src/application/components/Cache/Cache.tsx

```diff
@@ -70,20 +84,61 @@ export function Cache() {
               value={searchTerm}
             />
             <div className="overflow-auto h-full">
-              <EntityList
-                data={filteredCache}
-                selectedCacheId={cacheId}
-                setCacheId={setCacheId}
-                searchTerm={searchTerm}
-              />
+              <List>
```

Comment:
This component was fairly small and "EntityList" wasn't a super great name for this anyways, so I went ahead and inlined the implementation here and removed the old component.